src/main.c

```diff
@@ -357,13 +370,18 @@ main (int argc, char *argv[])
     }
   }
 
-  if (purgeYes != 0 && purge_after == 0)
+  if (purgeYes && !purge_after)
```

Comment:
I removed your edits from this part.

The change you see is just removing some operators. Unrelated to your patch.

Do you understand why you didn't need to make changes to it?